### PR TITLE
Change http to https for security links

### PR DESCRIPTION
**Pull request by huynq0911**
For security, we should change http into https links.

### CODE REVIEW

**Comment by dcbw** [comment]
sure...

File: plugins/meta/bandwidth/README.md [hunk 1]
```diff
@@ -59,6 +59,6 @@ Both egressRate and egressBurst must be set in order to limit egress bandwidth.
 
 ## tc tbf documentation
 
-- [tldp traffic control](http://tldp.org/HOWTO/Traffic-Control-HOWTO/components.html)
+- [tldp traffic control](https://tldp.org/HOWTO/Traffic-Control-HOWTO/components.html)
```
Comment (matthewdupre) [inline plugins/meta/bandwidth/README.md]: Amusingly this site has an incorrect certificate.  Perhaps we shouldn't change it?

Comment (huynq0911) [reply plugins/meta/bandwidth/README.md]: hmm, it can be accepted as long as the URL is not harmful.

Comment (huynq0911) [reply plugins/meta/bandwidth/README.md]: I have updated. Please check it!

**Pull request closed without merging**